library/src/lib/menu/menu.component.ts

```diff
@@ -12,9 +15,24 @@ import {
     encapsulation: ViewEncapsulation.None,
     styleUrls: ['menu.component.scss']
 })
-export class MenuComponent {
+export class MenuComponent extends AbstractFdNgxClass {
     /** @hidden */
     @HostBinding('class.fd-menu')
     fdMenuClass: boolean = true;
 
+    /** The separator line for each menu item. When set to true at list level, it adds a separator below each menu item in the list. 
+     * False by default. Leave empty for default. */
+    @Input()
+    separator: boolean = false;
+
+    /** @hidden */
+    constructor(public itemEl: ElementRef) {
+        super(itemEl);
+    }
+    /** @hidden */
+    _setProperties(): void {
+        if (this.separator) {
+            this._addClassToElement('fd-menu__list-separator');
```

Comment:
Also this class shouldn't be added there. We need to add class `fd-menu__list--separated` into `fd-menu-list` component.

Reply:
done